ui_testing/pages/order_page.py

```diff
@@ -39,6 +39,12 @@ def get_material_type(self):
 
     def get_article(self):
         return self.base_selenium.get_text(element='order:article').split(' No')[0]
+    def get_test_plan(self):
+        test_plans = self.base_selenium.get_text(element='order:test_plan')
+        if "×" in test_plans:
+            return test_plans.replace("× ", "").split('\n')
+        else:
+            return []
```

Comment:
this function is already defined in line https://github.com/modeso-open-source/1lims-automation/blob/832199fc4323809049d05d5ce6bcf407f2084cdb/ui_testing/pages/order_page.py#L78

Reply:
@dianaeldemerdash could u remove the duplicate code?

Reply:
Done